<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUser</class>
 <widget class="QWidget" name="AddUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#040609;</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>200</y>
     <width>721</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">			background-color: #10161f;
            color: white;
            border-radius: 18px;
            font-size: 16px;
           border: none;  </string>
   </property>
   <widget class="QLineEdit" name="lineEdit_phone">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>نام کاربری</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addUser">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>530</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">          background-color: #45aef5 ;
            color: white;
            border-radius: 10px;
            font-size: 16px;
           border: none;  </string>
    </property>
    <property name="text">
     <string>ثبت</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_code">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>170</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>نام کامل</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_phone">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>350</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>تلفن</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>رمز عبور</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_widget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Dana&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>افزودن کاربر</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_menu">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>940</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Dana&quot;;
background-color: rgb(255, 38, 49);
color: rgb(0, 0, 0);
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string>بازگشت به منو</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
